docs/reference/cat/templates.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[cat-templates]]
+== cat templates
+
+`templates` shows information about currently configured templates.
+
+[source, sh]
+--------------------------------------------------
+% curl 'localhost:9200/_cat/templates?v=true'
+name       template order version
+template0 te*      0
+template1 tea*     1
+template2 teak*    2     7
+--------------------------------------------------
+
+The output shows that we have three templates active on the cluster,
```

Comment:
three templates active -> three existing templates ? Templates cannot be active or deactivated